src/components/cuda_pw/common_defs.c

```diff
@@ -0,0 +1,106 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "common_defs.h"
+#include "debug_comp.h"
+#include "htable.h"
+
+#define ADDED_EVENTS_INITIAL_CAPACITY 64
+
+int initialize_dynamic_event_list_size(event_list_t * evt_table, int size)
```

Comment:
Unless you plan to use this elsewhere declare it `static`.

Reply:
Agree. Coming soon.

Reply:
have not made this static.